### PR TITLE
Update timeline to end of 2014

### PR DESCRIPTION
**Pull request by xiongchiamiov**
The last thing currently on our timeline is Obama's AMA in 2012; since we're now into 2015, we should probably update a bit. ;)

Ismail did a bunch of work gathering things up to mid-2014, but since #40 has somewhat died, I took its commits and incorporated them into this new PR.

I went back through recent blog posts and selected things that seemed to me relatively important.  There are two things I'm concerned about with the current state:
1. **There's a distinct lack of culture points.**  While I've been an active redditor for a while, I've stayed away from mainstream reddit until a few months ago, so I have very little idea what important things have happened there.
2. **It's visually crammed.**

image

I think this is due to an increase in the rate of things we're doing, rather than an over-liberal determination of things that should be on the timeline, but maybe not.  Either way, it looks a little bit crappy, and as we continue to do lots of stuff and we have to fit more and more time into the same amount of space, it will just get worse.  Perhaps we should, separately from this PR, make the timeline scroll or something.

:eyeglasses: @umbrae because you opted in to #40; feel free to opt-out here.

### CODE REVIEW

**Comment by umbrae** [comment]
:eyeglasses: @powerlanguage 

I think Josh will be a good asset both in terms of culture points and also on whether the design is beginning to break down. From the screenshot, I think we could probably sneak in this last round of changes before a new iteration is necessary, but I agree we're stretching.

**Comment by powerlanguage** [comment]
Agree with Chris' assessment that the current design can probably handle this round of changes, but will need revising before growing further.

**Suggestions for culture:**

April Fools events tend to get referred to on the site a lot:

* [time reddits](https://www.reddit.com/r/timereddits)
* [reddit gets hats](http://www.redditblog.com/2013/04/reddit-buys-team-fortress-2.html)
* [headdit](http://www.redditblog.com/2014/03/headdit-revolutionary-new-way-to-browse.html)

Others:

* [Banana for scale](https://www.reddit.com/r/WTF/comments/1qkd9x/secret_staircase_reveals_terrifying_secret/) - The meme originally started elsewhere, but it is ubiquitous on reddit now, and this was its introduction.
* [Foods over rice](https://www.reddit.com/r/AskReddit/comments/2np694/what_tasty_food_would_be_distusting_if_eaten_over) - Perhaps to soon to immortalize?
* [Time traveller AMA](https://www.reddit.com/r/IAmA/comments/9z4d5/10_years_from_now_i_am_a_scientist_working_at_the/)

I didn't follow the FCC stuff, but could be a good item as it had a lot of community support.

**Comment by xiongchiamiov** [comment]
It looks like we already have in this PR those three April Fools' posts, but they aren't `important`ed (made to display by default, rather than just on mouseover).  I did that and added the others you mentioned.

:haircut:

**Comment by powerlanguage** [comment]
:fish:

**Comment by umbrae** [comment]
:fish: thanks for doing this!

File: reddit_about/data/timeline.json [hunk 1]
```diff
@@ -9,17 +9,35 @@
...
+    {"date": "2014-07-23", "class": "org", "title": "reddit live launched", "url": "http://www.redditblog.com/2014/07/announcing-reddit-live.html"},
+    {"date": "2014-09-02", "class": "org", "title": "AMA app launched", "url": "http://www.redditblog.com/2014/09/announcing-official-reddit-ama-app_2.html"},
+    {"date": "2014-09-08", "class": "org", "title": "full-site HTTPS support", "url": "http://www.redditblog.com/2014/09/hell-its-about-time-reddit-now-supports.html"},
+    {"date": "2014-09-30", "class": "org important", "title": "outside funding", "url": "http://www.redditblog.com/2014/09/fundraising-for-reddit.html"},
+    {"date": "2014-10-15", "class": "org", "title": "Alien Blue acquired", "url": "http://www.redditblog.com/2014/10/the-alien-has-landed.html"},
+    {"date": "2014-11-13", "class": "org important", "title": "new CEO and return of kn0thing", "url": "http://www.redditblog.com/2014/11/coming-home.html"},
+    {"date": "2014-11-28", "class": "culture", "title": "foods over rice", "url": "https://www.reddit.com/r/WTF/comments/1qkd9x/secret_staircase_reveals_terrifying_secret"},
```
Comment (umbrae) [inline reddit_about/data/timeline.json]: Actually, this link is wrong.

**Comment by umbrae** [comment]
Whoops, :nail_care: actually.

**Comment by xiongchiamiov** [comment]
Whoops, looks like I didn't copy when I thought I did.  :haircut:

**Comment by umbrae** [comment]
:fish:

**Comment by xiongchiamiov** [comment]
7bc34b0 f8ae54a

**Pull request closed without merging**